misk-hibernate/src/main/kotlin/misk/jdbc/DataSourceConfig.kt

```diff
@@ -41,8 +41,9 @@ data class DataSourceConfig(
   val username: String? = null,
   val password: String? = null,
   val fixed_pool_size: Int = 10,
-  val connection_timeout: Duration = Duration.ofSeconds(30),
-  val connection_max_lifetime: Duration = Duration.ofMinutes(30),
+  val connection_timeout: Duration = Duration.ofSeconds(10),
+  val validation_timeout: Duration = Duration.ofSeconds(3),
+  val connection_max_lifetime: Duration = Duration.ofMinutes(1),
```

Comment:
we should probably set the `idleTimeout` as well, since we are changing the `maxLifetime`